packages/catalog-server/data/bootstrap-packages.json

```diff
@@ -1,40 +1,5 @@
 {
   "packages": [
-    "@shoelace-style/shoelace",
```

Comment:
Intentionally removed most of the bootstrap components?

Reply:
nope, oops!